### PR TITLE
Feat/asadmin timeout

### PR DESCRIPTION
**Pull request by conoyes**
fulfills realityforge/chef-glassfish/issues/75

### CODE REVIEW

File: providers/deployable.rb [hunk 1]
```diff
@@ -164,9 +165,10 @@ def archives_dir
 
     bash "asadmin_undeploy #{new_resource.component_name}" do
       unless cache_present
-        only_if "#{asadmin_command('list-applications')} #{new_resource.target}| grep -- '#{new_resource.component_name} '", :timeout => 150
+        only_if "#{asadmin_command('list-applications')} #{new_resource.target}| grep -- '#{new_resource.component_name} '", :timeout => node['glassfish']['asadmin']['timeout'] * 1000
```
Comment (realityforge) [inline providers/deployable.rb]: I believe the timeout should be in seconds here so can just remove the * 1000

File: providers/deployable.rb [hunk 2]
```diff
@@ -195,8 +197,9 @@ def archives_dir
   command << new_resource.component_name
 
   bash "asadmin_disable #{new_resource.component_name}" do
-    only_if "#{asadmin_command('list-applications --long')} #{new_resource.target} | grep '#{new_resource.component_name} ' | grep enabled", :timeout => 150
-    timeout 150
+    only_if "#{asadmin_command('list-applications --long')} #{new_resource.target} | grep '#{new_resource.component_name} ' | grep enabled", :timeout => node['glassfish']['asadmin']['timeout'] * 1000
```
Comment (realityforge) [inline providers/deployable.rb]: I think the * 1000 can be removed here

File: providers/deployable.rb [hunk 3]
```diff
@@ -210,8 +213,9 @@ def archives_dir
   command << new_resource.component_name
 
   bash "asadmin_enable #{new_resource.component_name}" do
-    not_if "#{asadmin_command('list-applications --long')} #{new_resource.target} | grep #{new_resource.component_name} | grep enabled", :timeout => 150
-    timeout 150
+    not_if "#{asadmin_command('list-applications --long')} #{new_resource.target} | grep #{new_resource.component_name} | grep enabled", :timeout => node['glassfish']['asadmin']['timeout'] * 1000
```
Comment (realityforge) [inline providers/deployable.rb]: And here

**Comment by conoyes** [comment]
Had to do a little merging due to previous pull request accepts and conflicts...

File: providers/deployable.rb (hunk 1 above)
Comment (conoyes) [inline providers/deployable.rb]: Oh right, correcting it asap. Good one.

File: providers/deployable.rb (hunk 2 above)
Comment (conoyes) [inline providers/deployable.rb]: Oh right, correcting it asap. Good one.

File: providers/deployable.rb (hunk 3 above)
Comment (conoyes) [inline providers/deployable.rb]: Yep.

**Comment by conoyes** [comment]
@realityforge I corrected the issues You pointed out

**Comment by realityforge** [comment]
Merged - thanks again!

**Pull request merged**